### PR TITLE
Added filter processor

### PR DESCRIPTION
**Pull request by margirova**

### CODE REVIEW

File: bspump/filter/filter.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+from ..abc.processor import Processor
+import asab
+
+###
+
+L = logging.getLogger(__name__)
+
+###
+
+class Filter(Processor):
```
Comment (ateska) [inline bspump/filter/filter.py]: It shouldn't be an abstract class.

**Comment by ateska** [comment]
Nice.
I suggest to also add "Include" fields ... both "Delete" and "Include" will apply only if the member is not None.

**Comment by margirova** [comment]
I thought it should work like if delete_fields is True, then it deletes all fields from event specified in lookup, otherwise it deletes only fields specified in lookup

**Comment by margirova** [comment]
That's what it does. However I see the naming is misleading

**Comment by ateska** [comment]
Well, you know what to do with incorrect naming.

**Comment by ateska** [comment]
What do you mean by inverse?

**Comment by ateska** [comment]
It should be "include" or "exclude" name.

**Pull request merged**